features/facebookImport/test/ministory/about-pictures-data-analysis.test.js

```diff
@@ -18,6 +18,10 @@ describe("Pictures ministory on export with no JPEG pictures in correct location
             "messages/inbox/user_2j57v4wtoa/photos/14567839_1208576379157271_232556476_n_21487783899157271.jpg",
             ""
         );
+        zipFile.addNamedEntry(
```

Comment:
Maybe you wanted to use `posts/media` here? Also, possibly test the `jpeg` extension, just in case?

Reply:
I wanted `photos/data` as that test checks that the story is not active if photos are found in other places.

Reply:
I updated one of the tests to test for `jpeg`